Adds running an RPC using systemd article (#104)
moondance-labs/tanssi-docs: .snippets/text/node-operators/appchain-node/fetching-bootnode-section.md

### Fetching Bootnode Information {: #fetching-bootnode-information}

> Bootnode information can be read from the Tanssi Appchain storage on its [Polkadot.js explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank}.

Reviewer: ```suggestion
Bootnode information can be read directly from Tanssi itself. For example, you can use the [Polkadot.js explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank} to get the bootnodes for a specific Appchain in the [Dancebox TestNet](/builders/tanssi-network/networks/dancebox/overview/){target=\_blank.
```

> Bootnode information can be read directly from Tanssi itself. For example, you can use the [Polkadot.js explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank} to get the bootnodes for a specific Appchain in the [Dancebox TestNet](/builders/tanssi-network/networks/dancebox/overview/){target=\_blank.

Reviewer 2: ```suggestion
Bootnode information can be read directly from Tanssi itself. For example, you can use the [Polkadot.js explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank} to get the bootnodes for a specific Appchain in the [Dancebox TestNet](/builders/tanssi-network/networks/dancebox/overview/){target=\_blank}.
```

To do so, take the following steps:

> 1. Select `dataPreservers` as the module to query
> 2. Set the storage query to `bootNodes`

Reviewer 2: ```suggestion
1. Select **dataPreservers** as the module to query
2. Set the storage query to **bootNodes**
```

3. Provide your Tanssi Appchain ID
4. Click on the **+** sign

> ![Getting the bootnode](/images/node-operators/appchain-node/rpc-docker-systemd-1.webp)

Reviewer 2: I have to zoom in 3x to see this image, although I am getting old so it could just be a me problem 😂
